docs/application/dotnet/guides/nui/layer.md

```diff
@@ -39,12 +51,41 @@ layer.Add(_imageView);
 
 ### Re-ordering layers
 
-A range of functions are provided to change the draw order of the layers.
+The default root layer obtained from the Window instance has a **Depth** field value of 0, which is the same as for a newly created layer. Adding a layer to the window using `window.AddLayer()` API increase the Depth value as shown in the following code example:
```

Comment:
@nawab-samsung Would you please check the grammar of this sentence?
It seems the "increase" should be "increases", that I missed to check at my previous comment.

Reply:
@mihashco 
You seem to have missed this as @mijin85cho pointed out.
```suggestion
The default root layer obtained from the window instance has a **Depth** field value of 0, which is the same as for a newly created layer. Adding a layer to the window using `window.AddLayer()` API increases the Depth value as shown in the following code example:

Reply:
Sure, i changed it in newest commit